site/en/docs/extensions/mv3/intro/mv3-migration/index.md

```diff
@@ -355,106 +339,109 @@ chrome.tabs.executeScript({
 ```
 
 ```js
-// Manifest v3
+// Manifest V3
 
 // background.js
+async function getCurrentTab() {/* ... */}
+let tab = await getCurrentTab();
+
 function showAlert() {
-  alert("test!");
+  alert('Function test alert');
 }
 
 chrome.scripting.executeScript({
+  target: {tabId: tab.id},
```

Comment:
This seems like the only practical change in this file (the rest is just reformatting).

Reply:
Just force pushed to remove the line wrap commit. This should make the meaningful changes more obvious.